Rs 711 (#649)
elasticpath/react-pwa-reference-storefront: src/components/src/ProductDisplayAttributes/README.md

# ProductDisplayAttributes

#### Description

> Displays product attribute information with the ability of folding.  The component takes an array of cortex product attributes with an added key `isOpened` which dictates whether or not the component should fold the attribute text.  An example of this structure can be seen in the MockHttpREsponses folder in this component.  Also refer to `productdisplayitem.main.tsx` for a real life example of how it is used.

Reviewer: ```suggestion
Displays product attribute information with the ability of folding. The component takes an array of cortex product attributes with an added key `isOpened` which dictates whether or not the component should fold the attribute text.  An example of this structure can be seen in the MockHttpResponses folder of this component. Also refer to `productdisplayitem.main.tsx` for an example of its usage.
```

Author: looks good to me
